src/home/urls.py

```diff
@@ -2,11 +2,14 @@
 from django.views.generic import TemplateView
 from django.utils.translation import gettext_lazy as _
 
-from home.views import (HomeView, NewsletterConfirmView,
-                        NewsletterSuccessView, ADDNAOptin)
+from home.views import (HomeView, ContactView,
+                        NewsletterConfirmView, NewsletterSuccessView,
+                        ADDNAOptin)
 
 urlpatterns = [
     path('', HomeView.as_view(), name='home'),
+
+    path('contact/', ContactView.as_view(), name='contact'),
```

Comment:
Peut-être ajouter la balise i18n pour la cohérence globale, même si la traduction est effectivement identique ?

```suggestion
    path(_('contact/'), ContactView.as_view(), name='contact'),
```

Reply:
(ignoré car on ne fait plus l'effort de traduire)